### PR TITLE
ISPN-12358 Fix log ranges

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-12358

### CODE REVIEW

File: core/src/main/java/org/infinispan/util/logging/Log.java [hunk 1]
```diff
@@ -104,6 +105,7 @@
  * @since 4.0
  */
 @MessageLogger(projectCode = "ISPN")
+@ValidIdRange(min = 1, max = 1000)
```
Comment (jabolina) [inline core/src/main/java/org/infinispan/util/logging/Log.java:108]: There is an overlap in range with infinispan-commons. They aren't duplicating the IDs, though.

File: query-core/src/main/java/org/infinispan/query/core/impl/Log.java [hunk 2]
```diff
@@ -24,6 +34,188 @@ public interface Log extends BasicLogger {
...
+//   void detectedUnknownIndexedEntity(String cacheName, String className);
+
+//   @LogMessage(level = WARN)
+//   @Message(value = "Found undeclared indexable types in cache %s : %s. No indexes were created for these types because autodetection is not enabled for this cache.", id = 14029)
+//   void detectedUnknownIndexedEntities(String cacheName, String classNames);
+
+//   @Message(value = "The type %s is not an indexed entity.", id = 14030)
+//   IllegalArgumentException notAnIndexedEntityException(String typeName);
+
+   @Message(value = "Unable to resume suspended transaction %s", id = 14033)
```
Comment (jabolina) [inline query-core/src/main/java/org/infinispan/query/core/impl/Log.java:114]: Just a nit. Move these to be in ID order.

File: core/src/main/java/org/infinispan/util/logging/Log.java (hunk 1 above)
Comment (tristantarrant) [reply core/src/main/java/org/infinispan/util/logging/Log.java:108]: I've renumbered the offending logs in core and adjusted the ranges. 
I know, renumbering isn't ideal, but it's a handful of messages and it simplifies our life in the long run.

**Comment by jabolina** [comment]
Seems a few related failures and we're good.

**Comment by tristantarrant** [comment]
I've fixed the failures

**Comment by jabolina** [comment]
Green jenkins and no related flaky.

**Comment by jabolina** [comment]
Thanks, @tristantarrant!

**Pull request merged**